server/models/db/workerTimeslot.js

```diff
@@ -21,7 +21,19 @@ async function updateWorkerAvailability(workerId, specificDate, newStatus) {
     return db.any(`update ${Tables.workerTimeslot} set status='${newStatus}' where date='${specificDate}' and worker_id='${workerId}';`);
 }
 
+async function checkWorkerAvailability(workerTimeslotId) {
+    //Selects all from workerTimeslot table 
+    const queryStatement = `select * from ${Tables.workerTimeslot}  where worker_timeslot_id = ${workerTimeslotId}`;
+    const queryOutput = await db.any(queryStatement);
+    //check if workertimeSlot is present 
+    if (_.isEmpty(queryOutput)) return false;
+    //check if available 
+    const currentStatus = _.map(queryOutput, 'status'); 
+    return _.includes(currentStatus, TimeslotStatus.available).
```

Comment:
Hi Nancy,
I would recommend removing the period at the end of this line and replacing it with a semicolon to prevent runtime issues.

Reply:
Nice catch! Done!